ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -320,7 +330,13 @@ const Annotations = React.memo(({ variant, mainGeneId, showMainGene }) => {
       {mainGeneId && <VariantGenes mainGeneId={mainGeneId} showMainGene={showMainGene} variant={variant} />}
       {(mainGeneId && Object.keys(variant.transcripts || {}).length > 1) && <VerticalSpacer height={10} />}
       <LargeText>
-        <b><UcscBrowserLink variant={variant} includeEnd={!!variant.svType} /></b>
+        <b><UcscBrowserLink variant={variant} includeEnd={!!svType && !endChrom} /></b>
+        {endChrom && (
+          <span>
+            ;&nbsp
+            <b><UcscBrowserLink variant={variant} isEnd /></b>
```

Comment:
put this entire section inside the `b` tag above, it is better for the semi colon to be bolded too

Reply:
okay